Gaijinizer patch/CommonEvents_643.txt

```diff
@@ -1,208 +1,266 @@
 ﻿> GAIJINIZER PATCH FILE
 > BEGIN STRING
 \\N[1]は、モルモーに犯されてしまった！！
-> CONTEXT: CommonEvents_643/13/Dialogue < UNTRANSLATED
-> CONTEXT: Troops/319/pages/0/37/Dialogue < UNTRANSLATED
-
+> CONTEXT: CommonEvents_643/13/Dialogue
+> CONTEXT: Troops/319/pages/0/37/Dialogue
+\\N[1] is being raped by Mormo!!#Ron
 > END STRING
 
 > BEGIN STRING
 ふふふ…\\N[1]、かわいい…♡
 
 そんなトロンとした表情しちゃって…♡
-> CONTEXT: CommonEvents_643/40/Dialogue/enemy25 < UNTRANSLATED
+> CONTEXT: CommonEvents_643/40/Dialogue/enemy25
+Fufufu, you're so cute. ♡
 
+You've got such a slack look on your face.#Ron
 > END STRING
 
 > BEGIN STRING
 普段は首筋からがメインだから滅多にアピールしないけど
 
 こっちに意識を集中させての正統派な搾精も得意なんだ♡
-> CONTEXT: CommonEvents_643/44/Dialogue/enemy25 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_643/44/Dialogue/enemy25
+I usually go for the neck so I don't often show
+myself off like this. But this kind of orthodox
+squeezing method where I get you to focus on me
+like this is also my specialty. ♡#Ron
 > END STRING
```

Comment:
Minor rephrasing:



**<!--StartFragment-->I usually go for the neck, so I don't often show
myself off like this. But this kind of orthodox
squeezing method where I get you to focus on
me is also my specialty. ♡<!--EndFragment-->**